new-pipeline/dist.html

```diff
@@ -47,16 +47,13 @@ <h2 style="font-size: 18px; margin-bottom: 0">
                     <select id="filter-os" class="multiselect" title="Target OS" data-all-selected="any OS" data-n-selected="OSs" multiple></select>
                     <select id="filter-arch" class="multiselect" title="Product architecture" data-all-selected="any architecture" data-n-selected="architectures" multiple></select>
                     with
-                    <select id="filter-e10s" class="multiselect" title="e10s setting" data-all-selected="any e10s setting" multiple></select>
```

Comment:
This also looks like you included some changes from another PR here.
It's probably easiest to follow Chris instructions in #419 first.